src/components/QueryEditor/EditorRows.tsx

```diff
@@ -1,15 +1,16 @@
 import React from 'react';
-
-import { Stack } from './Stack';
+import { EditorStack } from './EditorStack';
 
 interface EditorRowsProps {
   children?: React.ReactNode;
 }
-
+/**
+ * Uses Stack component from grafana-ui. Available starting from grafana-ui@10.2.3
+ */
```

Comment:
I think we don't need these comments, we have switched to grafana packages v10 and I think it should be fine.